Misc/NEWS.d/next/Library/2024-07-16-00-01-04.gh-issue-99631.GWD4fD.rst

```diff
@@ -0,0 +1 @@
+The :mod:`shelve` now accepts custom serializer and deserializer functions.
```

Comment:
```suggestion
The :mod:`shelve` module now accepts custom serialization
and deserialization functions.
```

(Not entirely sure, but I would say that 'serializer' is a serialization function but not a serializer function)